Create new config parameter phonebookid

BadDoom opened the pull request:
Create config parameter that will allow users to select different
phonebooks. For example Google synced phonebooks.

kaikreuzer commented on bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/FritzboxTr064Binding.java:
```diff
@@ -127,9 +131,14 @@ public void activate(final BundleContext bundleContext, final Map<String, Object
         this._pw = fboxpw;
         this._user = fboxuser;
         this._url = fboxurl;
+        this._pbid = Integer.valueOf(fboxphonebookid);
```
You should better catch NumberFormatExceptions here (and set 0 as a fallback).

kaikreuzer commented on bundles/binding/org.openhab.binding.fritzboxtr064/.classpath:
```diff
@@ -1,19 +1,19 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Why did that file now change with this commit? Could you remove it from the PR?

kaikreuzer commented on bundles/binding/org.openhab.binding.fritzboxtr064/.project:
```diff
@@ -25,8 +25,14 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```
there should be no m2e nature added - can you remove that from the PR?

kaikreuzer commented on bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/FritzboxTr064Binding.java:
```diff
@@ -138,7 +138,13 @@ public void activate(final BundleContext bundleContext, final Map<String, Object
+        
+        try{
+            this._pbid = Integer.valueOf(fboxphonebookid);
+    	}catch(NumberFormatException ex){ //set fallback to 0
```
Could you apply the auto-formatter to insert spaces around the curly brackets so that this matches the style of the rest of the code?

kaikreuzer commented on bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/FritzboxTr064Binding.java:
```diff
@@ -138,7 +138,13 @@ public void activate(final BundleContext bundleContext, final Map<String, Object
+            this._pbid = Integer.valueOf(fboxphonebookid);
+    	}catch(NumberFormatException ex){ //set fallback to 0
+            this._pbid = 0;
+
```
remove empty line here

BadDoom commented:
Hi Kai, sorry for uploading the classpath and project file, must have slipped in :-)
I reverted the last commit and uploaded now only the changes from the sources.
Also changed the format and added the requested spaces
